src/misc/utils.ts

```diff
@@ -157,7 +157,13 @@ export class Utils {
     static getHostname(uriString: string): string {
         const url = Utils.getUrl(uriString);
         try {
-            return url != null && url.hostname !== '' ? url.hostname : null;
+            let hostname = url != null && url.hostname !== '' ? url.hostname : null;
+
+            if (hostname != null && url.port !== '') {
+                hostname += ":" + url.port;
+            }
+
+            return hostname;
```

Comment:
This is what `getHost()` below is for. It should include the port already.

Reply:
@kspearrin , any objection changing the various clients' `view` UI to use `getHost()` instead of `getHostname()` for display purposes?

Reply:
Yes, that would be the proper fix for the above-linked issue.

Reply:
Reverted change. Made call to getHost.